src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -620,16 +620,13 @@ uct_rc_mlx5_ep_connect_qp(uct_rc_mlx5_iface_common_t *iface,
                           uct_ib_mlx5_qp_t *qp, uint32_t qp_num,
                           struct ibv_ah_attr *ah_attr)
 {
-    switch (qp->type) {
-    case UCT_IB_MLX5_OBJ_TYPE_VERBS:
-        return uct_rc_iface_qp_connect(&iface->super, qp->verbs.qp, qp_num, ah_attr);
-    case UCT_IB_MLX5_OBJ_TYPE_DEVX:
+    uct_ib_mlx5_md_t *md = ucs_derived_of(iface->super.super.super.md, uct_ib_mlx5_md_t);
+
+    if (md->flags & UCT_IB_MLX5_MD_FLAG_DEVX) {
```

Comment:
this change mans that if we have a global devx support we use devx modify qp even for verbs objects right?

Reply:
right